### ISSUE TITLE
Open file: bad error message

### ISSUE THREAD

**Issue by piegamesde** [nushell/nushell#1426]
**Is your feature request related to a problem? Please describe.**
When I do `open matrix.csv` and that file has some syntax errors in it, I get:

```
- shell:1:18
1 | open matrix.csv
  | ^^^^ input cannot be parsed as CSV
- shell:1:18
1 | open matrix.csv
  |      ------------- value originates from here
```

**Describe the solution you'd like**
I'd like to have a better error message, that tells me why my file is invalid and where exactly.

**Comment by poulp** [comment]
@piegamesde What is the content of your csv ? I tried to reproduce the problem with an invalid csv but the command does not fail and display nothing

**Comment by piegamesde** [comment]
I really don't know how exactly I produced this. If I get this error again, I'll let you know.

> but the command does not fail and display nothing

That's not any better, to be honest.

**Comment by Sosthene-Guedon** [comment]
I am able to reproduce the error with 
`open file.txt |from-csv `
In this context I managed to add small error message indicating the reason and the line of the csv error.

**Comment by Sosthene-Guedon** [comment]
This issue should  probably be closed, it was fixed by PR #1490
